File: Easy/Array/1684-Count-the-Number-of-Consistent-Strings/solution.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        vector<bool> alpha(26, false);
        int cnt = words.size();

        for(char ch : allowed) alpha[ch - 'a'] = true;

        for(auto word : words){
            for(char ch : word){
                if(!alpha[ch - 'a']){
                    cnt--;
                    break;
                }
            }
        }

        return cnt;
    }
};

File: Easy/Array/1684-Count-the-Number-of-Consistent-Strings/solution.py
class Solution(object):
    def countConsistentStrings(self, allowed, words):
        """
        :type allowed: str
        :type words: List[str]
        :rtype: int
        """

        cnt = len(words)
        for word in words:
            for ch in word:
                if ch not in allowed:
                    cnt -= 1
                    break

        return cnt